examples/tutorials/Part 08 bis - Introduction to Protocols.ipynb

```diff
@@ -1,408 +0,0 @@
-{
```

Comment:
👍

Someday, we'll get to write a new one! 😄

Reply:
Haha, totally! I guess I'll have to do it if I'm the one who removed the first version? 😅

Reply:
Let’s draw straws? 😆